selenium/src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,102 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BaseChromeTest;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise1Test extends BaseChromeTest {
+
+    public SoftAssertions softly = new SoftAssertions();
+    private static final int ITEMS_IN_HEADER = 4;
+    private static final int NUMBER_OF_IMAGES = 4;
+    private static final int NUMBER_OF_ITEMS_IN_LEFT_SIDE_BAR = 5;
+
+    @Test(groups = {"HW2"})
+    public void exercise1() {
```

Comment:
It's usually better to give meaningful names to test that describe what they're testing.